src/components/navlink/NavLink.js

```diff
@@ -1,7 +1,11 @@
 import "./NavLink.css";
 
-function NavLink() {
-  return <a>Future Nav Icon</a>;
+function NavLink(props) {
+  return (
+    <a>
+      <img className="navIcon" src={props.src} />
```

Comment:
`alt` missing!

Reply:
Used a Button element instead and added an aria-label.

Reply:
Good 👍 however, even if you use an `aria-label` you still need the `alt` attribute! In that case you can just set the alt attribute to `alt=""` just empty 🙂 Otherwise screenreaders would read out the link in `src` - and that's not what you want.